python/lsst/ts/salobj/cli.py

```diff
@@ -0,0 +1,35 @@
+# This file is part of ts_salobj.
```

Comment:
Please delete this file. It is no longer being used.

Reply:
Done